app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Api::V1::CommentsController < ApplicationController
+  def index
+    @post = Post.find_by_id(params[:post_id])
+    @comments = @post.comments
+    render json: @comments
+  end
```

Comment:
- [ ] If I hit this API `/api/v1/users/2/posts` then give me the following error :point_down: 

![Screenshot from 2023-08-05 02-01-44](https://github.com/anbehindY/Dev-Blog/assets/45226848/00d2767f-18a1-467e-8560-0cdd1e699a90)

- [ ] Kindly fix this error and please test your API endpoints on Postman :slightly_smiling_face: